OAPIF_explained.md

```diff
@@ -15,23 +15,23 @@ Since October 2024, a change has been made to the Geostandards on the [Dutch "Ap
 This list now includes the new generation of standards: OGC API Features and OGC API Tiles. The WMS and WFS profiles have been moved to the list of recommended standards.
 
 OAPIF has been considered as the successor of the OGC WFS standard, but that does not mean it will replace it in the near future, although eventually it might.
-At this moment, they are complementary to each other. Where WFS is mainly known and used in the GIS community, the OAPIF is aiming at the non GIS-community, like web developers. 
+At this moment, they are complementary to each other. Where WFS is mainly known and used in the GIS community, the OAPIF is aiming broader at both the GIS- and non GIS-community, like web developers. 

```

Comment:
Gezien de implementatie van OGC API in clients kan je stellen dat de OGC API zich ook op GIS gebruikers richt, naast developers.

Reply:
Verwerkt!